tee-worker/omni-executor/rpc-server/src/server.rs

```diff
@@ -73,7 +73,12 @@ pub async fn start_server<
 	jwt_secret: String,
 ) -> Result<(), Box<dyn std::error::Error>> {
 	let address = format!("0.0.0.0:{}", port);
-	let server = Server::builder().build(address.parse::<SocketAddr>()?).await?;
+	let max_connections: u32 =
+		env::var("RPC_SERVER_MAX_CONNECTIONS").unwrap_or("100".to_string()).parse()?;
```

Comment:
Should we prefix all env variables with `OE` or `OMNI_EXECUTOR` ?

Reply:
It sounds good